src/boards/bluemicro_nrf52833/pinconfig.c

```diff
@@ -8,8 +8,8 @@ const uint32_t bootloaderConfig[] =
   CFG_MAGIC0, CFG_MAGIC1,                       // magic
   5, 100,                                       // used entries, total entries
 
-  204, 0x100000,                                // FLASH_BYTES = 0x100000
-  205, 0x40000,                                 // RAM_BYTES = 0x40000
+  204, 0x80000,                                // FLASH_BYTES = 0x100000/2=0x80000
+  205, 0x20000,                                 // RAM_BYTES = 0x40000/2=0x20000
```

Comment:
the comment does not need to have the /2= just simply state the value for 833. Please also correct the comment column alignment as well.